GOVERNMENT.md

```diff
@@ -0,0 +1,56 @@
+
+The meeting should result in the authorship of a brief report.
+
+**Section 5.** Steering Committee Reports. The Steering Committee must author a brief report corresponding following each of its required biannaual meetings. This report must document the actions and considerations of the committee, and must be archived and disseminated via the GitHub repository. The report must include the names of the Steering Committee's members.
+
+**Section 6.** GitHub Repository Role. Steering Committee members must have the role of `Admin` on the GitHub repository.  
+
+**Section 7.** Specification Releases. The Steering Committee must unanimously approve of official releases and publish them via GitHub. Releases must be thoroughly documented and properly formatted as described in RELEASE.md. The Committee is subsequently responsible for alerting the community to the release.
+
+**Section 8.** Maintainer Election. Any member of the Steering Committee may at any time approve the election of a Maintainer. The Steering Committee member must give the Maintainer the role of `Maintain`.
+
+## Article II. Maintainers
+
+**Section 1.** Maintainer Definition. Maintainers are responsible for directing and moderating changes and fixes to the specification and its repository. This includes the merging of pull requests, the management of issues, and the preparation and validation of pending official releases. Maintainers are expected to advance the specification towards the goals laid out by the steering committee.
+
+**Section 2.** Election. Maintainers may be nominated or self-nominated at any time. A member of the Steering Committee must approve their election. Maintainers may serve subsequent terms.
+
+**Section 3.** Terms. Maintainers serve 2 year terms. At the end of their term, they must step down or seek the renewal of their title from a member of the Steering Committee.
+
+**Section 4.** Maintainers' Report. Maintainers must collectively author a biannual report summarizing the prominent actions and considerations of the Maintainers since the previous report. This report must include any critical issues or pending releases which the Steering Committee must consider. 
+
+**Section 5.** GitHub Repository Role. Maintainers must have the role of `Maintainer` on the GitHub repository.
+
+## Article III. Additional Bylaws
+
+**Section 1.** Pull Requests. Maintainers are ultimately responsible for the merging of pull requests into the working branches of the SNIRF specification, but both Steering Committee Members and Maintainers have the authority to do so. A Steering Committee members and Maintainers may NOT merge their own pull requests, but should instead request the review of another contributor.
+
+**Section 2.** Scheduling of Meetings. Meetings may be scheduled and organized by anyone. No fewer than 4 months and no more than 7 months should elapse between meetings such that they remain biannual.
```

Comment:
I would remove "No fewer than 4 months and". Why prevent meetings from happening more frequently if they are needed?

Reply:
Agreed